src/main/java/uk/gov/hmcts/cmc/claimstore/services/document/DocumentUploadException.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.cmc.claimstore.services.document;
+
+public class DocumentUploadException extends RuntimeException {
```

Comment:
To avid 500 error in logs, please add appropriate resolver in this class.
https://github.com/hmcts/cmc-claim-store/blob/master/src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/advices/ResourceExceptionHandler.java#L47